tests/bins_test.py

```diff
@@ -553,3 +553,32 @@ def test_bin_outer_and_inner_of_2d_without_event_coord():
     da.bins.coords['ynew'] = sc.bins_like(da, y)
     table = da.bins.constituents['data']
     assert sc.identical(result.hist(), table.hist(xnew=xnew, ynew=ynew))
+
+def test_bins_slicing_open_start_proper_stop_given():
```

Comment:
I just found that the existing test for this are in `test/slice_bins_test.py`. Can you move these there? I think your first 2 added tests may then be redundant, but please double check.